packages/shared/args.js

```diff
@@ -0,0 +1,30 @@
+function args(args) {
```

Comment:
`args(["hello", -c"])` returns `{c: undefined}` I propose this condition should throw an Error.